scripts/github_runner/Dockerfile.github_runner

```diff
@@ -0,0 +1,16 @@
+FROM myoung34/github-runner:ubuntu-bionic
+
+RUN apt-get update && \
```

Comment:
the cleaning script reports that `ifconfig` is missing. please add a package

Reply:
`scripts/setup_system.sh` has the required "net-tools" package.